packages/app/src/components/DashboardContainer.tsx

```diff
@@ -22,14 +21,16 @@ export function DashboardContainer(props: DashboardContainerProps) {
         (e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => setSearch(e.target.value),
         [],
     )
-    const systemMode = useSystemPreferencePalette()
+    const setThemeMode = useSetThemeMode()
+    const mode = useThemeMode()
+
     useEffect(() => {
```

Comment:
If there is a flickering between switching themes during startup, you can deal with it with `useLayoutEffect`.